### ISSUE TITLE
Merge physical and digitised Sierra works

### ISSUE THREAD

**Issue by jtweed** [wellcometrust/platform#1008]

**Comment by alexwlchan** [comment]
This is about aggregating Sierra data with itself (nothing to do with Miro!).

In some cases, we have separate bibs with:

* Items which are physical objects
* Items which are digitised copies of the physical things

We want to be able to combine these two bibs into a single Work, and ensure that both bibs point to the same Work in our API.

**Comment by wilfox** [comment]
I wrote these rules back in April but not sure if I shared them:

Rules for determining the print version of an E-book:

Use the contents of "Marctag":"776 w" from all "materialType" "e-book" bib records.

If the contents of subfield w contains the MARC organisation code (UkLW) in parentheses ignore this part of the code and use the remainder of the code, which starts with the letter b, to search for an exact match against "materialType" "Books" bibliographic record numbers. An exact match will identify the print counterpart to the E-book.

Example:

(UkLW)b21414440 ignore (UkLW) and use the remainder of the code: b21414440 to search for an exact match against other bibligraphic record numbers.

If the contents of subfield w contains the MARC organisation code (OCoLC) in parentheses use the entire code, including the code in parentheses and search for an exact match against the contents of "Marctag" : "035" in all "materialTye" "Books" bibliographic records.  An exact match will identify the print counterpart to the E-book.

Example:

(OCoLC)14322288 use the entire code: (OCoLC)14322288 to search for an exact match against "Marctag" : "035" in other bibliographic records.
